webapp/channels/src/components/copy_url_context_menu/copy_url_context_menu.tsx

```diff
@@ -29,46 +29,52 @@ type Props = {
     };
 }
 
-export default class CopyUrlContextMenu extends React.PureComponent<Props> {
-    copy = () => {
-        let link = this.props.link;
+const CopyUrlContextMenu = ({
+    link,
+    siteURL,
+    actions,
+    menuId,
+    children,
+}: Props) => {
+    const copy = useCallback(() => {
+        let siteLink = link;
```

Comment:
why do we have to store this into a new variable?

Reply:
Because If we directly change the `link` variable it will break the `useCallback` as `link` is added to the dependency list of useCallback. It will also violate the eslint rule `no-parram-reassign`.